.github/scripts/env.sh

```diff
@@ -4,6 +4,8 @@ if [ "$1" != "nightly_wheel" ];then
     source /opt/intel/oneapi/compiler/latest/env/vars.sh
     source /opt/intel/oneapi/umf/latest/env/vars.sh
     source /opt/intel/oneapi/pti/latest/env/vars.sh
+    source /opt/intel/oneapi/ccl/latest/env/vars.sh
+    source /opt/intel/oneapi/mpi/latest/env/vars.sh
```

Comment:
For building, you don't need it. In runtime, I think you will use `pip install` instead of `source`.